packages/react/src/internal/hooks/use-id.ts

```diff
@@ -0,0 +1,15 @@
+import { useMemo } from 'react';
+
+function makeid(length = 5) {
+  let result = '';
+  const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+  const charactersLength = characters.length;
+  for (let i = 0; i < length; i += 1) {
+    result += characters.charAt(Math.floor(Math.random() * charactersLength));
+  }
+  return result;
+}
+export function useId() {
+  const id = useMemo(() => makeid(), []);
+  return id;
+}
```

Comment:
How do we guarantee these IDs to be unique?


```suggestion
function createIdGenerator() {
  let counter = BigInt(0); 
  return () => (counter++).toString(36);
}

const  makeId = createIdGenerator();

export function useId() {
  const id = useMemo(() => makeId(), []);
  return id;
}
```

Btw, 62^5 is 916132832.